packages/auto-approve/src/check-config.ts

```diff
@@ -135,12 +135,12 @@ export async function checkAutoApproveConfig(
       } else {
         // This branch means auto-approve is not on this repo, so we're
         // throwing an error (essentially, skipping the check)
-        throw AutoApproveNotConfigured;
+        throw new AutoApproveNotConfigured();
       }
     } catch (err) {
       // This branch means auto-approve is not on this repo, so we're
       // throwing an error (essentially, skipping the check)
-      throw AutoApproveNotConfigured;
+      throw new AutoApproveNotConfigured();
```

Comment:
Can we add a test to make sure this is being appropriately caught?

Reply:
added!